client/containers/ConfirmationModalContainer/index.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { Modal, Button } from 'react-bootstrap'
+import { connect } from 'react-redux'
+import * as actions from '../../actions'
+
+export function ConfirmationModalComponent({ show, title, body, handleHide, handleAction }) {
+    const action = () => (
+        Promise.resolve(handleAction())
+        .then(handleHide)
+    )
+    return (
+        <Modal show={show} onHide={handleHide}>
+            <Modal.Header>
+                <a className="close" onClick={handleHide}>
+                    <i className="icon-close-small" />
+                </a>
+                <h3>{ title }</h3>
+            </Modal.Header>
+            <Modal.Body>
+                { body }
```

Comment:
Are we planning to have only text displayed in the modal body? Can we use a ReactElement as body object as well ? Because we might want to use some React components in the confirmation. Example would be to use RelatedPlanning component while deleting events.